icd/api/vk_compute_pipeline.cpp

```diff
@@ -328,8 +328,9 @@ VkResult ComputePipeline::Create(
         binaryCreateInfo.pipelineFeedback.feedbackValid = true;
         binaryCreateInfo.pipelineFeedback.duration = duration;
         pDefaultCompiler->SetPipelineCreationFeedbackInfo(
-                pPipelineCreationFeadbackCreateInfo,
-                &binaryCreateInfo.pipelineFeedback);
+            pPipelineCreationFeadbackCreateInfo, nullptr,
```

Comment:
each parameter should be on its own line here

Reply:
done